src/devices_models/devices/common/add_to_expression.jl

```diff
@@ -1011,7 +1011,8 @@ function add_to_expression!(
         add_expressions!(container, T, devices, model)
     end
     expression = get_expression(container, T(), V)
-    for d in devices, mult in get_expression_multiplier(U(), T(), d, W())
+    for d in devices
+        mult in get_expression_multiplier(U(), T(), d, W())
```

Comment:
Is this line doing anything?

Reply:
looks like a comma is missing

Reply:
fixed in #1104 (this is a weird thing that seems to have been there for years